package.json

```diff
@@ -26,7 +26,7 @@
     "posttest": "istanbul check-coverage"
   },
   "dependencies": {
-    "harvest": "git+ssh://git@github.com:holidayextras/harvest.git#v2.0.0",
+    "harvest": "git+ssh://git@github.com:holidayextras/harvest.git#WEB-11095",
```

Comment:
ill change it later, i leave it pointing to the branch till before i merge to master making it easier for to pull and test the main branch